### PR TITLE
Feature: Components

### PR DESCRIPTION
**Pull request by luftedar**
# Feature: Components
In this update:
- Added components dir
- Added calculator.js
- Updated UI of the calculator.
- Used class-based components

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,22 +1,14 @@
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
```
Comment (rloterh) [inline src/App.js:4]: - [ ] Your application is not starting due to **improper Eslint configuration**. Kindly ensure to improve upon the same. You can optionally install  clean linter from [Microverse Eslint](https://github.com/microverseinc/linters-config/tree/master/react-redux)

**Eslint configuration errors:**
image

image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Orcun, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (dasileker) [reply src/App.js:4]: - [ ] the issue still persists, please search on the internet how to fix it.

Capture web_23-11-2021_1469_localhost

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @luftedar,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luftedar** [comment]
Hi @dasileker. I am not receiving this issue on my side this is the problem.

**Comment by dasileker** [comment]
Some time it happens, but the issues is that you need to add npm install airbnb-base to your dependencies

**Comment by luftedar** [comment]
Thanks I installed them hope it will work :+1:

**Review by AshakaE — APPROVED** [review]
### Hi @luftedar   🙋‍♀️, 

# STATUS APPROVED ✔

Thanks for making the requested changes 👏 
   
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**